requirements/requirements.in

```diff
@@ -5,6 +5,8 @@ msgpack~=0.6.1
 pymysql
 Pygments
 Rtree~=0.9.4
+sphinx-autobuild
+sphinx-rtd-theme
```

Comment:
I think these belong in a separate file (`requirements/docs.in`) since sphinx is only required to build the docs and isn't required in the final build artifacts. Probably `requirements/dev.in` should include `requirements/docs.in` so devs can easily build docs.